ggml/src/ggml.c

```diff
@@ -15199,10 +15198,10 @@ static void ggml_compute_forward_flash_attn_ext_f16(
     const float m0 = powf(2.0f, -(max_bias       ) / n_head_log2);
     const float m1 = powf(2.0f, -(max_bias / 2.0f) / n_head_log2);
 
-    enum ggml_type    const k_vec_dot_type = type_traits[k->type].vec_dot_type;
-    ggml_from_float_t const q_to_vec_dot   = type_traits[k_vec_dot_type].from_float;
-    ggml_vec_dot_t    const kq_vec_dot     = type_traits[k->type].vec_dot;
-    ggml_to_float_t   const v_to_float     = type_traits[v->type].to_float;
+    enum ggml_type    const kq_vec_dot_type = type_traits[k->type].vec_dot_type;
+    ggml_from_float_t const kq_from_float   = type_traits[kq_vec_dot_type].from_float;
```

Comment:
My personal opinion is that the old names were better because `kq_` as a prefix is ambiguous but I don't really care either way.

Reply:
Thanks for taking a look - changed back to the original names